common/src/main/java/com/example/android/uamp/media/UampPlaybackPreparer.kt

```diff
@@ -54,9 +54,14 @@ class UampPlaybackPreparer(
             PlaybackStateCompat.ACTION_PREPARE_FROM_MEDIA_ID or
                     PlaybackStateCompat.ACTION_PLAY_FROM_MEDIA_ID or
                     PlaybackStateCompat.ACTION_PREPARE_FROM_SEARCH or
-                    PlaybackStateCompat.ACTION_PLAY_FROM_SEARCH
+                    PlaybackStateCompat.ACTION_PLAY_FROM_SEARCH or
+              PlaybackStateCompat.ACTION_PREPARE
 
-    override fun onPrepare() = Unit
+    override fun onPrepare() {
+        Log.d(TAG, "Received a prepare command")
+        // TODO: Make this non-manual
+        onPrepareFromMediaId("wake_up_01", null)
```

Comment:
We should do this properly before merging. Thoughts on using `SharedPrefs` to store the most recently played item?

Reply:
Done!